### PR TITLE
BossRank

### PR DESCRIPTION
**Pull request by BosssRank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
 ![IMG_20240124_233523](https://github.com/erenkarakal/SkCheese/assets/67760502/29000f28-f504-40b8-81a2-f52c070572ea)
 
-### SkCheese is a Skript addon that adds new language features which will likely not make it to official Skript. Please check the wiki for more info.
+---
+
+# SkCheese
+SkCheese is a [Skript](https://github.com/SkriptLang/Skript/) addon . This adds the feature's to skript which cannot add to Skript plugin officially , because SkBee  is deleted now so SkCheese has become a good replacement
```
Comment (BosssRank) [inline README.md:6]: forgot .

**Comment by erenkarakal** [comment]
There are grammar errors in this PR. SkCheese is not an SkBee replacement and will never be. The plugin.yml authors are for people who worked on the plugin's code and not other stuff and I don't plan on updating the readme anytime soon.

**Comment by BosssRank** [comment]
I think this change is very good and helps the begginers, because now skbee (addon every one was using) is deleted . . .

**Comment by BosssRank** [comment]
coders should see good description when install this addon to make it more popular

**Pull request closed without merging**